include/aspect/heating_model/adiabatic_heating.h

```diff
@@ -54,6 +54,13 @@ namespace aspect
     {
       public:
         /**
+         * Returns whether we compute a simplified adiabatic heating term by
+         * neglecting dynamic pressure effects.
+         */
+        bool
+        use_simplified_adiabatic_heating() const;
```

Comment:
I don't really know what this means in formulas. It would be great to include the two forms of the term in the documentation, depending on whether or not  the simplified form is used.